app/controllers/box_requests_controller.rb

```diff
@@ -1,5 +1,5 @@
 class BoxRequestsController < ApplicationController
-  before_action :set_box_request, only: [:show, :edit, :update, :destroy]
+  before_action :set_box_request, only: %i[show edit update destroy]
```

Comment:
I appreciate all the small cleanups you've added in this PR. I won't comment on every one other than to say thanks.